### DISCUSSION TITLE
Erro na instalação do Git

### DISCUSSION THREAD

**Discussion by matheusbastosnhoncanse** [learningssik/pull-request#114] [General]
Ajuda
Quando estava no final da instalação do Git apareceu essa mensagem:
![codigo do git](https://github.com/learningssik/Get-new-badges/assets/153521707/cd3796d9-fdc7-407c-b5db-8b36ad363ec9)
Mas instalação continuou normal e quando foi configurar apareceu essa mensagem:
![codig de erro](https://github.com/learningssik/Get-new-badges/assets/153521707/1ad4e7ce-3809-466f-a5cc-4e84ca38fc10)
Alguém pode me ajudar?

**Comment by learningssik** [comment]
Recommend removing the git and reinstalling it.
